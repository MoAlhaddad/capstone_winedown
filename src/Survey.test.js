import React from "react";
import Survey from "./components/Survey";
import renderer from "react-test-renderer";

describe("Survey", () => {
  it("rendered Survey", () => {
    const component = renderer.create(<Survey />);
    expect(component).toMatchSnapshot();
  });
});
