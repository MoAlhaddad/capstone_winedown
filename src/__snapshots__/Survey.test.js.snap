// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Survey rendered Survey 1`] = `
<div>
  <div
    className="question-section"
  >
    <div
      className="question-count"
    >
      <span>
        Question 
        1
      </span>
      /
      3
    </div>
    <div
      className="question-text"
    >
      Where do you prefer your wine coming from?
    </div>
  </div>
  <div
    className="answer-section"
  >
    <button
      className="btn btn-primary"
      disabled={false}
      onClick={[Function]}
      style={
        Object {
          "margin": "0 10px",
        }
      }
      type="button"
    >
      Spain
    </button>
    <button
      className="btn btn-primary"
      disabled={false}
      onClick={[Function]}
      style={
        Object {
          "margin": "0 10px",
        }
      }
      type="button"
    >
      France
    </button>
    <button
      className="btn btn-primary"
      disabled={false}
      onClick={[Function]}
      style={
        Object {
          "margin": "0 10px",
        }
      }
      type="button"
    >
      Italy
    </button>
    <button
      className="btn btn-primary"
      disabled={false}
      onClick={[Function]}
      style={
        Object {
          "margin": "0 10px",
        }
      }
      type="button"
    >
      other
    </button>
  </div>
</div>
`;
